lib/babe/babe.go

```diff
@@ -297,7 +262,9 @@ func (b *Service) Stop() error {
 
 // Authorities returns the current BABE authorities
 func (b *Service) Authorities() []types.Authority {
-	return b.epochData.authorities
+	auths := make([]types.Authority, len(b.epochHandler.epochData.authorities))
+	copy(auths, b.epochHandler.epochData.authorities)
+	return auths
```

Comment:
I don't think you can copy `types.Authority` directly, you would need a deep copy method I guess?

Reply:
done